go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/containernetworking/cni
 go 1.14
 
 require (
+	github.com/containernetworking/plugins v0.9.1
```

Comment:
I wondered if we could add another `go.mod` in the `plugins/debug` directory to avoid adding this dependency at top level.

Reply:
I agree with you. Changed.